db/re/item_db_equip.yml

```
@@ -23314,6 +23335,24 @@ Body:
     Script: |
       bonus5 bAutoSpellWhenHit,"NPC_MAGICMIRROR",6,150,BF_MAGIC,0;
       bonus bUnbreakableShield;
+  - Id: 2197
+    AegisName: Iron_Shield
+    Name: Iron Shield
+    Type: Armor
+    View: 4
+    Weight: 2800
+    Defense: 80
+    Slots: 1
+    Locations:
+      Left_Hand: true
+    ArmorLevel: 1
+    EquipLevelMin: 90
+    Refineable: true
+    Script: |
+      bonus bUnbreakableShield;
+      bonus bAspdRate,-3;
+      if (getrefine()>=8)
+         bonus bNoKnockback;
```

For me the malus bAspdRate should be when refine is >= 8
```suggestion
      if (getrefine()>=8) {
         bonus bAspdRate,-3;
         bonus bNoKnockback;
      }
```

---

Thanks !